### DISCUSSION TITLE
Pointing to the latest release of an action

### DISCUSSION THREAD

**Discussion by pantelis-karamolegkos** [community/community#38142] [Actions] [answered]
Is there a way to make actions always pointing to the latest release?

Does `v0` work for all actions as for example it does for the gcp auth [action](https://github.com/google-github-actions/auth)?

**Comment by swrobel** [comment]
I just tried all of the following, and none of them worked:
* Removing the `@v` specifier from my actions
* Trying `@v0` for all of my actions
* Trying `@latest` for all of my actions

It would be great if there was a way to actually do this

**Comment by airtower-luna** [comment, marked as answer]
There's nothing built-in, the part after the `@` is just a ref (branch or tag name) or commit hash. Many action developers provide a `vN` branch or sliding tag (not exactly best practice in my opinion) for stable releases with a given major version. The closest you can get would be to use the default branch (e.g. `@main`), but that might result in random breakage if the developers add changes that aren't backwards compatible.

> Does v0 work for all [action](https://github.com/google-github-actions/auth)s as for example it does for the gcp auth action?

Actually it doesn't, it's just a tag that points at the same commit as the latest tag in the v0.x series: https://github.com/google-github-actions/auth/tags

Personally I'd pin a version or hash and configure Dependabot for updates. That way you get all updates, but not in surprising ways. :wink:
